Makefile

```diff
@@ -1,7 +1,7 @@
 test:
 	py.test
 testcov:
-	py.test --cov=ideascube/ --cov-report=term-missing
+	py.test --cov=ideascube/ --cov-report=term-missing --migrations
```

Comment:
We may want to remove the explicit run of `migrate` from travis, then, no?

https://github.com/ideascube/ideascube/blob/master/.travis.yml#L12